src/app/loadout-builder/process-worker/process-utils.ts

```diff
@@ -273,9 +262,9 @@ export function canTakeSlotIndependantMods(
               assignments[sortedItems[i].id].push(raidMod.hash);
             }
           }
-
-          return true;
         }
+
+        return true;
```

Comment:
This is the bug fix. Everything else is just cleanup/simplification.

Reply:
Awesome! Thanks for that. This one slipped past me due to some weirdness in beta.